sdk/typescript/src/types/objects.ts

```diff
@@ -7,18 +7,39 @@ import {
   Infer,
   number,
   enums,
-  unknown,
   union,
   tuple,
   object,
+  any,
+  literal,
 } from 'superstruct';
+import {
+  SuiAddress,
+  SuiAddressSchema,
+  TransactionDigest,
+  TransactionDigestSchema,
+} from './common';
 
 export type ObjectDigest = Infer<typeof ObjectDigestSchema>;
 export type ObjectId = Infer<typeof ObjectIdSchema>;
 export type SequenceNumber = Infer<typeof SequenceNumberSchema>;
 export type ObjectRef = Infer<typeof ObjectRefSchema>;
+export type ObjectContent = Infer<typeof ObjectContentSchema>;
+export type ObjectOwner =
```

Comment:
Why this cannot be inferred from [ObjectOwnerSchema](https://github.com/MystenLabs/sui/pull/1510/files#diff-91e1be21f1ea53756e723bcdf46519bc214054bb68cf69db9b8bccd6c4578fdeR68)